policy/modules/system/locallogin.te

```diff
@@ -123,6 +123,7 @@ auth_manage_pam_runtime_dirs(local_login_t)
 auth_manage_pam_runtime_files(local_login_t)
 auth_manage_pam_console_data(local_login_t)
 auth_domtrans_pam_console(local_login_t)
+auth_use_pam_motd_dynamic(local_login_t)
```

Comment:
If this is a Debian-based image then you should probably build with ifdef distro_debian, which would allow this access
But even if you want to allow this unconditionally, then you should remove the distro_debian conditional as that then is redundant

Reply:
This is not a Debian based image, but an image generated by using Buildroot (see the PR description). My aim is to discuss how to handle such changes properly if they do not apply to all systems supported by the refpolicy.

I have one question though: do you know why so far Debian-based distributions alone need such a rule, and others do not?

Reply:
Well you called a pretty specific interface that to me suggested that your device was using the PAM motd-dynamic module (which is PAM module used by Debian)

Why exactly is that execute permission needed?

Reply:
Actually now i see it "sh" is mislabeled ....
find / -inum 416

it should be labeled shell_exec_t

once you fixed  the label you should be able to omit this commit

Reply:
Thanks! This fixed this issue.